src/main/kotlin/sliv/tool/model/project/AlgorithmOutput.kt

```diff
@@ -0,0 +1,6 @@
+package sliv.tool.model.project
+
+import java.nio.file.Path
+
+data class AlgorithmOutput(val kind: LayerKind, val path: Path, val containedLandmarks: List<Long>) {
```

Comment:
don't use algo output in naming), empty `{}` may be omitted, what is Path?

Reply:
no such class in scheme

Reply:
>import java.nio.file.Path

Reply:
no such class in scheme - is about `Algorithm output`